### ISSUE TITLE
M20

### ISSUE THREAD

**Issue by pq** [flutter/flutter-intellij#1502]
This is the plan for the M20 milestone.

### Android Studio
- [ ] doctor doesn't report a flutter plugin in Android Studio #1438

### Inspector
- [x] land [Flutter PR](https://github.com/flutter/flutter/pull/12932) to unblock IntelliJ Inspector @jacob314 
- [x] enable [flag](https://github.com/flutter/flutter-intellij/blob/master/src/io/flutter/sdk/FlutterSettingsConfigurable.java#L41)
- [x] continued polish and iteration @pq @jacob314 

### Hot reload
- [ ] wire into the flutter_tools protocol to get soft reload failure messages #1398

### Build and infra
- [x] continued infra improvements around build and release @stevemessick 

---

## Releasing

The release date is 12/6.

- [x] update change notes @pq
- [x] create candidate build @pq
- [ ] testing ([script](https://github.com/flutter/flutter-intellij/blob/master/docs/testing.md)):
  - [x] run test script on IntelliJ CE @pq
    - https://github.com/flutter/flutter-intellij/issues/1565
  - [ ] run test script on Android Studio windows @devoncarew 
  - [ ] run test script on Android Studio @stevemessick 
  - [ ] run test script on internal build @jwren
  - [ ] run test script on IntelliJ CE EAP @devoncarew
- [ ] publish plugin @pq
- [ ] send blog post @wmleler 

---

## Deferred

---

🏃 in progress, ✋ blocked, 💪 stretch
